examples/nifgen/basic-arbitrary-waveform.py

```diff
@@ -0,0 +1,171 @@
+# Demonstrates how to configure an NI Arbitrary Waveform Generator to generate a basic arbitrary waveform with Arbitrary Waveform Mode.
+# 
+# The gRPC API is built from the C API. NI-FGEN documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niFgen\documentation\English\SigGenHelp.chm
+#
+# Getting Started:
+#
+# To run this example, install "NI-FGEN Driver" on the server machine.
+# Link : https://www.ni.com/en-us/support/downloads/drivers/download.ni-fgen.html
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# 
+#
+# Running from command line:
+#
+# Server machine's IP address, port number, and resource name can be passed as separate command line arguments.
+#   > python basic-arbitrary-waveform.py <server_address> <port_number> <resource_name>
+# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763", with "SimulatedFGEN" as the resource name
+
+import grpc
+import sys
+import nifgen_pb2 as nifgen_types
+import nifgen_pb2_grpc as grpc_nifgen
+import nitclk_pb2 as nitclk_types
```

Comment:
Do you need tclk and session imports? I don't see it being used.

Reply:
removed that